caustic/__init__.py

```diff
@@ -1,12 +1,15 @@
 __version__ = '0.4.10'
 __author__ = "Ciela"
 
+from . import lenses, sources, utils, sims, demo
```

Comment:
So I did this because I did have an import conflict. I tried to import  something from `caustic.utils` but since we were populating the namespace with `from lenses import *` it was overloaded with `caustic.lenses.utils` and I couldn't access the stuff I needed. 

Also, this way of importing things is more future proof. If we keep adding more modules we are bound to get even more name conflicts as long as we are dumping everything into the same namespace. 

A compromise could be to fill the caustic namespace more intentionally. Instead of dumping every module, we could say the `lenses` and `sources` should be accessible, but not utils. So you'd still need to do `caustic.lenses.utils` to get lens utils and `caustic.utils` to get caustic utils, but `caustic.SIE` would work. It's a bit more work to pick out what you want, but its a common practice in large packages to take common functions and move them to a higher level namespace for easy access.